src/Classes/Canvas.ts

```diff
@@ -254,21 +256,9 @@ export class Canvas {
         const container = this.container;
         const options = container.options;
 
-        let radius: number;
-        let opacity: number;
         let colorValue: string | undefined;
-
-        if (particle.bubbler.radius !== undefined) {
-            radius = particle.bubbler.radius;
-        } else {
-            radius = particle.radius;
-        }
-
-        if (particle.bubbler.opacity !== undefined) {
-            opacity = particle.bubbler.opacity;
-        } else {
-            opacity = particle.opacity.value;
-        }
+        let radius = particle.bubbler.radius !== undefined ? particle.bubbler.radius : particle.radius;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'radius' is never reassigned; use 'const' instead of 'let'.](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5216054)